app/models/prisoner_validation.rb

```diff
@@ -1,6 +1,7 @@
 class PrisonerValidation
-  UNKNOWN            = 'unknown'.freeze
-  PRISONER_NOT_EXIST = 'prisoner_does_not_exist'.freeze
+  UNKNOWN                   = 'unknown'.freeze
+  PRISONER_NOT_EXIST        = 'prisoner_does_not_exist'.freeze
+  PRISONER_LOCATION_INVALID = 'prisoner_location_not_valid'.freeze
```

Comment:
The validation for location needs the `unknown` state. For example the prisoner could be found and then the location api could timeout. All of the other checks follow the pattern of having 3 states: valid, invalid and unknown